pkg/security/rconfig/policies.go

```diff
@@ -130,6 +144,15 @@ func (r *RCPolicyProvider) LoadPolicies(macroFilters []rules.MacroFilter, ruleFi
 	defer r.RUnlock()
 
 	load := func(id string, cfg []byte) {
+		if r.dumpPolicies {
+			name, err := writePolicy(id, cfg)
+			if err != nil {
+				log.Debugf("unable to dump the policy: %s", err)
```

Comment:
maybe something other than debug ? by default this won't be shown which can be surprising